native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -1,65 +1,101 @@
 import UIKit
 import SharedCode
 
-class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+class ViewController: UIViewController {
+    
+    @IBOutlet var departureStationScrollWheel: UIPickerView!
+    @IBOutlet var arrivalStationScrollWheel: UIPickerView!
 
-    @IBOutlet private var label: UILabel!
-    //@IBOutlet weak var tableview: UITableView!
+    @IBOutlet var fareSearchButton: UIButton!
     @IBOutlet var trainsTable: UITableView!
+
+    @IBAction func buttonPressed(_ sender: Any) {
```

Comment:
Could we give this function a more descriptive name please, i.e. trainSearchButtonPressed